docs/jwst/data_products/product_types.rst

```diff
@@ -1,81 +1,131 @@
 Data Product Types
 ------------------
-The following table contains a list of all data product types, as given by their file name suffix,
-the pipeline and stage of processing that creates each type, and an indication of whether the file naming
-is exposure-based or source-based. The product name suffixes are active links to detailed descriptions
-in the following sections.
+The following tables contain a list of all data product types, as given by their file name suffix. There is one table per stage of processing.
+All tables indicate whether the file naming is exposure-based (Exp) or source-based (Srce).
```

Comment:
I would drop the "e" from the end of "Srce". It just looks funny (at least to me) and most astronomers understand the abbreviation "src" (it also saves one letter's worth of space in the table).

Reply:
Done